src/ui/preview/preview-header.vue

```diff
@@ -21,12 +20,6 @@
                 :current-view="currentView"
                 @setView="setView"
             />
-            <NotebookMenuSwitcher v-if="showNotebookMenuSwitcher"
-                                  :domain-object="domainObject"
-                                  :ignore-link="true"
-                                  :object-path="objectPath"
```

Comment:
Since we dont need object-path any more, can we remove it from being injected into the vue file also

Reply:
removed other redundant code as well 😄